code/modules/screen_alert/command_alert.dm

```diff
@@ -12,6 +12,44 @@
 	style_open = "<span class='maptext' style=font-size:20pt;text-align:center valign='top'>"
 	style_close = "</span>"
 
+/obj/screen/text/screen_text/command_order/intro
```

Comment:
1) dont get why this is a command_order child
2) also should probably be renamed from /intro to something like /picture so it's clearer if/when we use it for something new